diffprivlib/tools/histograms.py

```diff
@@ -217,6 +230,10 @@ def histogramdd(sample, epsilon=1.0, bins=10, range=None, normed=None, weights=N
                       "specified for each dimension independently of the data (i.e., using domain knowledge).",
                       PrivacyLeakWarning)
 
+    if accountant:
+        check_accountant(accountant)
+        accountant.check_spend(epsilon, 0)
+
```

Comment:
I would change this code in the check_spend method of BudgetAccountant
This way we can have also the "no accountant" scenario

Reply:
As the argument now defaults to a `BudgetAccountant` instance, I have removed the `if()` statement, thereby simplifying the implementation. See b6ef353.